apps/www/_blog/2024-08-13-supabase-realtime-broadcast-and-presence-authorization.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Realtime Broadcast and Presence Authorization'
+description: 'Create Authorization policies for Realtime Broadcast and Presence'
+author: filipe
+image: 2024-08-13-supabase-realtime-broadcast-and-presence-authorization/OG.png
+thumb: 2024-08-13-supabase-realtime-broadcast-and-presence-authorization/thumb.png
+categories:
+tags:
+  - realtime
+  - engineering
+date: '2024-08-13'
+toc_depth: 2
+---
+
+Today we're releasing Realtime Authorization for Broadcast and Presence in Public Beta.
+
+For context, Supabase includes three useful extensions for building real-time applications.
+
+1. [Broadcast](https://supabase.com/docs/guides/realtime/broadcast): Send ephemeral, low-latency messages between users.
+2. [Presence](https://supabase.com/docs/guides/realtime/presence): Show when users are online and share state between users.
+3. [Postgres Changes](https://supabase.com/docs/guides/realtime/postgres-changes): Listen to Postgres database changes.
+
+This release introduces authorization for Broadcast and Presence using Row Level Security policies:
+
+```sql
+create policy "authenticated user can listen to all messages"
+on "realtime"."messages"
+as permissive for select
+to authenticated
+using ( true );
+```
+
+To facilitate authorization in Realtime, a new table `realtime.messages` is created automatically by Supabase Realtime when it connects to your Database. This table represents your [Channels](/docs/guides/realtime/concepts#channels) as objects against which you can write RLS policies.
+
+<Img
+  src={{
+    dark: '/images/blog/2024-08-13-supabase-realtime-broadcast-and-presence-authorization/schema--dark.png',
+    light:
+      '/images/blog/2024-08-13-supabase-realtime-broadcast-and-presence-authorization/schema--light.png',
+  }}
+  alt="The table that is used to test the policies."
+/>
+You can write Postgres RLS policies for this table and Realtime will use it them control the access to
+Broadcast and Presence for your channels based on the operations set in your policies: * `SELECT` policies:
+Able to receive messages from the channel * `INSERT` policies: Able to send messages to the channel
+
+This approach to control access should be familiar to users of [Supabase Storage](/docs/guides/storage/security/access-control).
```

Comment:
I don't think we need this. It's not really that similar (because we're not really storing anything in the db), and it's all RLS.

```suggestion
```